server/keycloak/addingCourse.go

```diff
@@ -17,9 +17,9 @@ func CreateCourseGroupsAndRoles(ctx context.Context, courseName, iterationName s
 
 	baseGroupName := fmt.Sprintf("%s-%s", courseName, iterationName)
 	subGroupNames := []string{
-		fmt.Sprintf("%s-Lecturer", baseGroupName),
-		fmt.Sprintf("%s-Editor", baseGroupName),
-		fmt.Sprintf("%s-Student", baseGroupName),
+		fmt.Sprintf("%s-%s", baseGroupName, CourseLecturer),
+		fmt.Sprintf("%s-%s", baseGroupName, CourseEditor),
+		fmt.Sprintf("%s-%s", baseGroupName, CourseStudent),
```

Comment:
Is it necessary to build it in each request? Or should this be outsourced as "global" variable?